### ISSUE TITLE
we should keep the 2 strategic goals about meeting members business needs and ensuring industry adoption. 

### ISSUE THREAD

**Issue by judy-zhuhr** [WebStandardsFuture/Vision#32]
We talked about the Strategic goals before, there were many very good descriptions in the previous version drafted by Avneesh. I think we should keep the 2 strategic goals about meeting members business needs and ensuring industry adoption. 
1.	Because W3C as a member organization, helping members to realize their business value in W3C is the key factor to keep the members. If the members stay, W3C can continue its work on Web standards. 
2.	Also, the industry adoption is the key factor to reflect the importance of W3C standards. If there is very little industry adoption, the standards work is probably failed. 
Since these 2 strategical goals are both about business and industry, I suggest we can merge it into one, such as “develop high quality Web standards to meet members business needs and ensure that major standards are adopted by the industry”.

**Comment by avneeshsingh** [comment]
Rephrasing it a little:
Ensure that the web standards and guidelines address the members business needs and facilitate their adoption by the industry.

**Comment by mnot** [comment]
I do not think this is part of our vision for the Web. It's merely a means to fund the Consortium; there may be others. I.e., it's tactical, not strategic.

**Comment by dwsinger** [comment]
yes, we need Vision to be a bright, desirable, imagined future: "We see a world in which everyone can use the web, in their own language, respecting their own culture, and according to their own abilities; using it without fear that they are being manipulated or misled, confident that their privacy is being respected, secure that they are not being abused. We see a world in which the web is used to inform, educate, support, and entertain; to underpin buying and selling, commerce and business. We see a web that is uniform across the world, where people of all kinds, ages, and conditions can meet and engage with each other. We are not there yet; there is much still to do." That's a vision. How we get there is really important, but it's different.

**Comment by judy-zhuhr** [comment]
I think we are talking about the strategy, not the very big vision. and then all of the things in this slide are not that kind of big etc. all are needed to change as you said in the meeting. here I think I agree with Aveneesh's changes on the wording. 
and also if we have time to change it more beautiful in this limited time, it is fine. but if we have so few limited time, I am afraid Avennesh may not have enough time.

**Comment by judy-zhuhr** [comment]
it has two parts, Vision and Strategy. this wording change is for the strategy part, not for Vision part
